File: RETOS/Reto04/Reto04.js
/*
 * Escribe un programa que se encargue de comprobar si un número es o no primo.
 * Hecho esto, imprime los números primos entre 1 y 100.
 */

function isPrimo(numero){
    if(numero<=1) return false;
    for (let i = 2; i <= Math.sqrt(numero); i++) {
        if(numero%i===0) return false;
    }
    return true;
}

console.log("Números primos entre 1 y 100:");
for (let i = 1; i <= 100; i++) {
    if(isPrimo(i)){
        console.log(i)
    }
}

File: RETOS/Reto04/Reto04.py
'''
Escribe un programa que se encargue de comprobar si un número es o no primo.
Hecho esto, imprime los números primos entre 1 y 100.
'''

import math

def isPrimo(numero):
    if numero<=1:
        return False
    
    for i in range(2, int(math.sqrt(numero))+1):
        if numero % i == 0:
            return False
    
    return True

print("Nùmeros primos entre 1 y 100:")

for i in range (1,101):
    if isPrimo(i):
        print(i)